paywall/src/components/lock/UnlockFlag.js

```diff
@@ -24,14 +24,16 @@ export const UnlockedFlag = () => {
     return () => clearTimeout(timeout)
   }, [setHidden])
   return (
-    <Flag id="UnlockFlag" hidden={hidden}>
+    <Flag hidden={hidden}>
```

Comment:
Generally I think it is better to use the styled components to set attributes which need to be set for all instances of a given component, rather than expect/hope that the implementer will mot forget to add the `id` or `className`.